packages/cli/src/commands/installedapps/delete.ts

```diff
@@ -1,12 +1,12 @@
 import { InstalledApp } from '@smartthings/core-sdk'
 
-import { SelectingAPICommand } from '@smartthings/cli-lib'
+import { selectAndActOn, APICommand } from '@smartthings/cli-lib'
 
 
-export default class InstalledAppDeleteCommand extends SelectingAPICommand<InstalledApp> {
+export default class InstalledAppDeleteCommand extends APICommand {
 	static description = 'delete the installed app instance'
 
-	static flags = SelectingAPICommand.flags
+	static flags = APICommand.flags
```

Comment:
`selectAndActOn` doesn't contribute any flags. I wasn't sure if I should include an empty map that could be added here for the sake of consistency or not. My biggest worry with leaving it out would be someone cutting and pasting a command that doesn't use it and then updating it to one that does and not remembering to the flags. What do you guys think?

Reply:
I think we are already relying on devs to remember to include flags for any new command even if not copying pasting, so this isn't some new risk. Adding an empty map is fine but would start cluttering the otherwise simple function, so I vote to leave it. We should clearly state somewhere that anyone building a new command will need to ensure flags are set accordingly.